src/components/bs5/banner/banner.stories.js

```diff
@@ -155,6 +155,17 @@ export const WithCallToAction = {
   },
 }
 
+/**
```

Comment:
add figma link to advanced banner
https://www.figma.com/design/qKsxl3ogIlBp7dafgxXuCA/QLD-GOV-DDS?node-id=7287-691763

cross link to https://www.designsystem.qld.gov.au/components/banners#section__advanced-banner